recipes-containers/kube-router/files/kube-router-watcher.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=kube-router restarter
+
+[Service]
+Type=oneshot
+ExecStart=/bin/systemctl restart kube-router.service
```

Comment:
What's the purpose of this? Why isn't having the kube-router.service restarting enough?

Reply:
There is a condition where identity.json gets re-created and all services need to be restarted.  Kube-rouer.path is defined but cannot restart anything itself.  According to the docs, a .path can only activate a unit file, not restart one.  

  https://superuser.com/questions/1171751/restart-systemd-service-automatically-whenever-a-directory-changes-any-file-ins

Reply:
If any service is dependent on identity.json then it should restart to take in the changes. Looks like kube-router needs deviceID so we need a watcher file to restart the service.